gitnamehere/point-of-sale-group-project
# Orders page

**Ticket:** [POS - Orders Page](https://cs375-pos-group-project.atlassian.net/jira/core/projects/ATS/board?selectedIssue=ATS-43)

**PR Desc:**
- Created a temporary frontend for the order page that displays the orders in a table
- - The orders are retrieved using a fetch at the URL `/api/orders`
- - The table is sorted via the order ID and so far only displays the total amount of items and total cost
- - Added placeholders for the date, customer, and status
- Added the feature where the user clicks on the order row, it will drop down a box of the order details
- - The order details are placed in a table that displays the item ID, name, quantity, and price
- - The item details are retrieved using a fetch at the URL `/api/items/${id}` where the id is the item ID
- Added a placeholder for the filter button that will be implemented in the future

**Testing:**
- Visit `http://localhost:3000/pos/ordering/`
- Add items to your order then click `Place Order`
- Repeat this for 2 or 3 orders
- Afterwards, visit `http://localhost:3000/pos/ordering/orderPage.html`
- Confirm that when you click on the order row it displays the items you ordered
